cinema/migrations/0002_movie_image.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 4.1 on 2024-10-29 16:23
+
+import cinema.models
```

Comment:
The import statement `import cinema.models` is incorrect for accessing the `movie_image_path` function. You should use `from cinema.models import movie_image_path` to directly import the function needed for the `upload_to` parameter.